### PR TITLE
Proper validation of RGBA arguments

### PR DESCRIPTION
**Pull request by CarlosFigueiraMSFT**
The first three arguments to the RGBA function are integer values, and as such are truncated when used - `RGBA(123.45, 0, 0, 1)` is the same as `RGBA(123, 0, 0, 1)`. However, the validation is done prior to the truncation, which makes some inputs to be considered invalid that should be valid, such as `RGBA(255.1, 0, 0, 1)`.

### CODE REVIEW

**Comment by LucGenetier** [comment]
✅ No public API change.

File: src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/RGBA.txt [hunk 1]
```diff
@@ -22,6 +22,9 @@ Error({Kind:ErrorKind.InvalidArgument})
 >> RGBA(255, 1.5, 255, 1.0)
 RGBA(255,1,255,1)
 
+>> RGBA(255.5, 1.5, 255.9, 1.0)
```
Comment (MikeStall) [inline src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/RGBA.txt:25]: Great test - Can you add a comment on this to explain it.

File: src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryColor.cs [hunk 2]
```diff
@@ -62,28 +62,29 @@ private static byte ParseColor(Match match, string color)
 
         public static FormulaValue RGBA(IRContext irContext, NumberValue[] args)
         {
+            // Truncate (strip decimals)
```
Comment (MikeStall) [inline src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryColor.cs:65]: With #1072, can we now do the truncation at the IR level? 

We'd just need to implement TexlFunction.GetArgPreprocessor - in theory, LibraryColor.cs wouldn't even need any changes.

**Review by MikeStall — APPROVED** [review]

**Comment by LucGenetier** [comment]
✅ No public API change.

File: src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryColor.cs (hunk 2 above)
Comment (CarlosFigueiraMSFT) [reply src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryColor.cs]: Updated function definition to use truncation at IR

**Pull request merged**